### PR TITLE
Merge branch 2.0 into master

### PR DESCRIPTION
**Pull request by kitsunet**

### CODE REVIEW

**Comment by kitsunet** [comment]
Note: This includes the merge from 1.2 into 2.0 already.

**Comment by kdambekalns** [comment]
:+1:

File: TYPO3.Neos/Configuration/Settings.yaml [hunk 1]
```diff
@@ -377,7 +377,7 @@ TYPO3:
 
     # Settings for the TYPO3 Neos Event Log (** DO NOT USE, NO PUBLIC API YET **)
     eventLog:
-      enabled: TRUE
+      enabled: FALSE
```
Comment (hlubek) [inline TYPO3.Neos/Configuration/Settings.yaml]: As discussed this should/could be enabled by a separate Commit for master

**Comment by hlubek** [comment]
:+1: Looks good to me
